components/net_traits/request.rs

```diff
@@ -488,6 +497,8 @@ pub struct Request {
     #[ignore_malloc_size_of = "Defined in rust-content-security-policy"]
     pub csp_list: Option<CspList>,
     pub https_state: HttpsState,
+    /// Servo internal: if crash details are present, trigger a crash error page with these details
```

Comment:
Ditto.

Reply:
fixed in c85c102311a8914f8a34f0fb6932be6b2f163382